### PR TITLE
Fix multiple axiom annotations

### LINKED ISSUE #35: Problem rendering multiple axiom annotations in the tree
This set of axiom annotations (with the same blank node subject) is not rendering properly in the tree:

https://github.com/ontodev/gizmos/blob/master/tests/resources/statements.tsv#L133

This is what I'm seeing, with some comments added:

```
<li>
  <span property="IAO:0000116">Entity doesn't have a closure axiom because the subclasses don't necessarily exhaust all possibilites. For example Werner Ceusters 'portions of reality' include 4 sorts, entities (as BFO construes them), universals, configurations, and relations. It is an open question as to whether entities as construed in BFO will at some point also include these other portions of reality. See, for example, 'How to track absolutely everything' at http://www.referent-tracking.com/_RTU/papers/CeustersICbookRevised.pdf</span>
  <small resource="_:riog00000256">
    <ul>
      <li>
        <div></div> <!-- Not sure what this is for. An empty subject? Should not be a div. -->
        <a href="?id=IAO:0010000">IAO:0010000</a>
      </li>
      <li>
        <a href="?id=rdfs:comment">rdfs:comment</a>
        <div></div>  <!-- the object should go here, but it should be a span -->
      </li>
      <li>
        <a href="?id=rdfs:seeAlso">rdfs:seeAlso</a>
        <div></div>   <!-- the object should go here, but it should be a span -->
      </li>
    </ul>
    <div hidden="true"><div></div><div></div><div></div></div> <!-- this seems very weird -->
  </small>
</li>
```

### PR DESCRIPTION
**Pull request by beckyjackson**
Resolves #35 

We needed to allow blank nodes in `row2o`. Before, if a blank subject was passed, it would just return `["div"]`, which is why we were seeing all those weird div tags.

### CODE REVIEW

File: gizmos/tree.py [hunk 1]
```diff
@@ -1101,10 +1081,10 @@ def renderOwlClassExpression(given_rows, rel=None):
 
 
 def row2po(stanza, data, row, db):
-    """Convert a predicate and object from a sqlite query result row to hiccup-style HTML."""
+    """Convert a predicate and object/value from a sqlite query result row to hiccup-style HTML."""
     predicate = row["predicate"]
     predicate_label = data["labels"].get(predicate, predicate)
-    p = ["a", {"href": curie2href(predicate, db)}, predicate_label]
+    p = ["b", ["a", {"href": curie2href(predicate, db)}, predicate_label]]
```
Comment (jamesaoverton) [inline gizmos/tree.py:1087]: I don't want the predicates to be bold.

**Review by jamesaoverton — CHANGES_REQUESTED** [review]
This makes the axiom annotations visible, which is what I asked for.

But I don't like how the axiom annotations are handled so differently from the top-level annotations. I'd prefer that they were the same: a nested list. And I suspect that the code could be recursive.

If that sounds hard, we could go with this for now.

File: gizmos/tree.py (hunk 1 above)
Comment (beckyjackson) [reply gizmos/tree.py:1087]: They blend in with the object, if the object is a CURIE or IRI. Is that OK?

Comment (jamesaoverton) [reply gizmos/tree.py:1087]: If it's a nested list, then it won't blend in.

**Comment by beckyjackson** [comment]
OK - I got a nested list working, not sure if there are any weird cases that might break this though:

image

And if you use multiple levels:

image

**Comment by beckyjackson** [comment]
I had to change how axiom annotation blank nodes were grabbed because it looks like second-level axiom annotations don't get the `rdf:type owl:Axiom` on them (see `_:riog00000003`)
```
SELECT * FROM statements;
rdf:type|_:riog00000001|rdf:type|owl:Ontology|||
OBI:123|OBI:123|rdfs:label||label|xsd:string|
OBI:123|OBI:123|rdf:type|owl:Class|||
OBI:123|_:riog00000003|rdfs:label||label|xsd:string|
OBI:123|_:riog00000003|owl:annotatedTarget||label|xsd:string|
OBI:123|_:riog00000003|owl:annotatedProperty|rdfs:label|||
OBI:123|_:riog00000003|owl:annotatedSource|_:genid2|||
OBI:123|_:genid2|rdfs:label||label|xsd:string|
OBI:123|_:genid2|owl:annotatedTarget||label|xsd:string|
OBI:123|_:genid2|owl:annotatedProperty|rdfs:label|||
OBI:123|_:genid2|owl:annotatedSource|OBI:123|||
OBI:123|_:genid2|rdf:type|owl:Axiom|||
OBI:123|_:riog00000003|rdf:type|owl:Annotation|||
```

**Review by jamesaoverton — APPROVED** [review]

**Pull request merged**